RecoParticleFlow/PFClusterProducer/interface/alpaka/PFClusteringEdgeVarsDeviceCollection.h

```diff
@@ -0,0 +1,16 @@
+#ifndef RecoParticleFlow_PFRecHitProducer_interface_alpaka_PFClusteringEdgeVarsDevice_h
+#define RecoParticleFlow_PFRecHitProducer_interface_alpaka_PFClusteringEdgeVarsDevice_h
+
+#include "DataFormats/Portable/interface/alpaka/PortableCollection.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+
+#include "RecoParticleFlow/PFClusterProducer/interface/PFClusteringEdgeVarsSoA.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE::reco {
+
+  using PFClusteringEdgeVarsDeviceCollection = PortableCollection<::reco::PFClusteringEdgeVarsSoA>;
```

Comment:
Could you copy here the comments from `PFClusteringEdgeVarsSoA` about how many elements should be used for these collections ?